eclipse-collections/src/main/java/org/eclipse/collections/impl/list/mutable/FastList.java

```diff
@@ -262,18 +263,33 @@ public int getBatchCount(int batchSize)
         return this.toArray((E[]) new Object[sourceToIndex - sourceFromIndex + 1], sourceFromIndex, sourceToIndex, 0);
     }
 
+    /**
+     * Overrides default method from List.
+     *
+     * @since 10.0
+     */
+    @Override
+    public void sort(Comparator<? super T> comparator)
+    {
+        if (this.size > 1)
```

Comment:
ComparableTimSort already has this optimization.

```java
        int nRemaining  = hi - lo;
        if (nRemaining < 2)
            return;  // Arrays of size 0 and 1 are always sorted
```

Reply:
Nice.  I will remove the optimization then.